### PR TITLE
Don't comment and close PRs that are already closed

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: migrate_issues.rb [hunk 1]
```diff
@@ -165,15 +169,12 @@ def hendl_pr(original)
...
-# names = @tools
-# source = "fastlane/playground" # TODO: Should be fastlane
-# names.each do |current|
-#   Hendl.new(source: "fastlane/#{current}",
-#        destination: "fastlane/fastlane",
-#             reason: "`fastlane` is now a mono repo, you can read more about this decision in our [blog post](https://fastlane.tools). All tools are now available in the [fastlane main repo](https://github.com/fastlane/fastlane).")
-# end
+require './tools'
+names = @tools
+names.delete("fastlane") # we don't want to import issues from our own repo
```
Comment (i2amsam) [inline migrate_issues.rb]: `names = @tools.reject { |tool| tool == "fastlane" }` better to copy than mutate in place if something might be shared.

File: migrate_issues.rb [hunk 2]
```diff
@@ -165,15 +169,12 @@ def hendl_pr(original)
...
+destination = "fastlane/playground" # TODO: Should be fastlane
+names.each do |current|
+  Hendl.new(source: "fastlane/#{current}",
+       destination: destination,
+            reason: "`fastlane` is now a mono repo, you can read more about this decision in our [blog post](https://krausefx.com/blog/our-goal-to-unify-fastlane-tools). All tools are now available in the [fastlane main repo](https://github.com/fastlane/fastlane).")
```
Comment (i2amsam) [inline migrate_issues.rb]: `change` rather than `decision`

**Comment by i2amsam** [comment]
:+1:

**Comment by KrauseFx** [comment]
Very good points, will submit separate PRs for that, thanks :+1:

**Pull request merged**